restapi/src/main/java/io/stargate/web/restapi/resources/v2/CqlResource.java

```diff
@@ -0,0 +1,120 @@
+package io.stargate.web.restapi.resources.v2;
+
+import static io.stargate.web.docsapi.resources.RequestToHeadersMapper.getAllHeaders;
+
+import com.codahale.metrics.annotation.Timed;
+import io.stargate.core.util.ByteBufferUtils;
+import io.stargate.db.ImmutableParameters;
+import io.stargate.db.Parameters;
+import io.stargate.db.datastore.ResultSet;
+import io.stargate.db.query.builder.BuiltQuery;
+import io.stargate.web.models.ApiError;
+import io.stargate.web.resources.Converters;
+import io.stargate.web.resources.RequestHandler;
+import io.stargate.web.restapi.dao.RestDB;
+import io.stargate.web.restapi.dao.RestDBFactory;
+import io.stargate.web.restapi.models.GetResponseWrapper;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import java.nio.ByteBuffer;
+import java.util.List;
+import java.util.Map;
+import java.util.function.UnaryOperator;
+import java.util.stream.Collectors;
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import javax.servlet.http.HttpServletRequest;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.QueryParam;
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import org.apache.cassandra.stargate.db.ConsistencyLevel;
+
+@Api(
+    produces = MediaType.APPLICATION_JSON,
+    consumes = MediaType.TEXT_PLAIN,
+    tags = {"data"})
+@Path("/v2/cql")
+@Produces(MediaType.APPLICATION_JSON)
+@Singleton
+public class CqlResource {
```

Comment:
Do we want to add this implementation for Stargate V1 as well? This will not be used or needed with Stargate V2.
We are planning on removing most of SGv1 REST implementation in couple of weeks (from `main` at least).
The only use case would be for Astra Classic I suppose.
But leaving this out would allow leaving out `QueryBuilderImpl` too.

Reply:
I assume we need it for classic,  should this be in old v2 or should I move to v1?

Reply:
@tjake Class to modify is correct for Stargate V1 but branch wrong. So would need separate PRs here.

@jeffreyscarpenter I think we need to talk about this, wrt our refactoring and so on.

Reply:
I removed the v1 code